### PR TITLE
LPD-43705 Increase error messages toast duration and move popover to be activated on hover instead of click

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPD-43705 ](https://liferay.atlassian.net/browse/LPD-43705)

### CODE REVIEW

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/SidePanelContent.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ export function saveAndReload() {
 }
 
 export function openToast(options: {
+	autoClose?: number;
```
Comment (cgmonte) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/SidePanelContent.tsx:30]: No need to upgrade this function, as we should directly use the one from `'frontend-js-web'`.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectDefinitionDetails.tsx [hunk 2]
```diff
@@ -163,7 +163,7 @@ export function RightSidebarObjectDefinitionDetails({
 			catch (error: unknown) {
 				const {message} = error as Error;
 
-				openToast({message, type: 'danger'});
+				openToast({autoClose: 15000, message, type: 'danger'});
```
Comment (cgmonte) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectDefinitionDetails.tsx:166]: Please `import {openToast} from 'frontend-js-web'` and use it directly.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectFieldDetails.tsx [hunk 3]
```diff
@@ -121,6 +121,7 @@ export function RightSidebarObjectFieldDetails() {
 			}
 			catch (error) {
 				openToast({
+					autoClose: 15000,
```
Comment (cgmonte) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectFieldDetails.tsx:124]: Please `import {openToast} from 'frontend-js-web'` and use it directly.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.tsx [hunk 4]
```diff
@@ -138,7 +138,7 @@ export function RightSidebarObjectRelationshipDetails({
 			catch (error: unknown) {
 				const {message} = error as Error;
 
-				openToast({message, type: 'danger'});
+				openToast({autoClose: 15000, message, type: 'danger'});
```
Comment (cgmonte) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.tsx:141]: Please `import {openToast} from 'frontend-js-web'` and use it directly.

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hey, I have requested a small change. Thanks!

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/SidePanelContent.tsx (hunk 1 above)
Comment (larissacribeiro) [reply modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/SidePanelContent.tsx]: I agree, done!

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectDefinitionDetails.tsx (hunk 2 above)
Comment (larissacribeiro) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectDefinitionDetails.tsx:166]: just did it for this and the other files, thank you!!

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by larissacribeiro** [comment]
ci:forward

**Comment by larissacribeiro** [comment]
ci:forward

**Pull request closed without merging**